### PR TITLE
Addition of information in Note

### PR DESCRIPTION
**Pull request by joinimran**
Add additional information in the Note section to enable policy to work.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8241

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/remote-credential-guard.md [hunk 1]
```diff
@@ -109,6 +109,8 @@ There are no hardware requirements for Windows Defender Remote Credential Guard.
 > [!NOTE]
 > Remote Desktop client devices running earlier versions, at minimum Windows 10 version 1607, only support signed-in credentials, so the client device must also be joined to an Active Directory domain. Both Remote Desktop client and server must either be joined to the same domain, or the Remote Desktop server can be joined to a domain that has a trust relationship to the client device's domain.
 
+> GPO [Remote host allows delegation of non-exportable credentials](https://docs.microsoft.com/windows/client-management/mdm/policy-csp-credentialsdelegation) should be enabled for Delegation of non-exportable credentials.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/remote-credential-guard.md:112]:
```suggestion
> GPO [Remote host allows delegation of non-exportable credentials](https://docs.microsoft.com/windows/client-management/mdm/policy-csp-credentialsdelegation) should be enabled for delegation of non-exportable credentials.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by joinimran** [comment]
Thanks @JohanFreelancer9 - @e0i Ready to go live.

**Review by DulceMontemayor — APPROVED** [review]

**Pull request merged**